addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -314,14 +323,40 @@ odoo.define('website.tour.form_editor', function (require) {
         {
             content:  "Wait reloading...",
             trigger:  "html:not(:has(#completlyloaded)) div",
+        },
+        // Check that if we edit again and save again the default value is not deleted.
+        {
+            content: 'Enter in edit mode again',
+            trigger: 'a[data-action="edit"]',
+            run: 'click',
+        },
+        {
+            content: 'Verify that the editor appeared',
+            trigger: 'button[data-action="save"]',
+            run: function () {}
+        },
```

Comment:
This does not need to be an extra step, at worst you should add it as an extra_trigger of the next step